src/plugins/vis_type_xy/public/components/xy_settings.tsx

```diff
@@ -152,9 +155,10 @@ export const XYSettings: FC<XYSettingsProps> = ({
   return (
     <Settings
       debugState={window._echDebugStateFlag ?? false}
+      onPointerUpdate={onPointerUpdate}
       xDomain={adjustedXDomain}
       rotation={rotation}
-      theme={[themeOverrides, theme]}
+      theme={[theme, themeOverrides]}
```

Comment:
This is flipped, the lower the index the higher the priority. So in this case `themeOverrides` is used only when the value does not exist on the `theme`.

Are you running into issue with this? There may be an issue where if a value is `undefined` on the first object this will not be overridden by the second.

Reply:
Interestingly, after the modifying, it began to work as expected. The same approach is used elsewhere, so if you're right the problem is everywhere

Reply:
Yeah. Also if the `theme` is a full theme, (i.e. defines all available property) that means the overrides are pointless and will never be used.